problem_builder/public/css/mentoring.css

```diff
@@ -62,6 +63,10 @@
     display: table; /* Enable line-wrapping in IE8 */
 }
 
+.mentoring legend p:last-child {
+    margin-bottom: 0 !important;
```

Comment:
We're very allergic to !important - is there a way you can write without this? !important overrides everything on the page and is strongly discouraged